command/commands/team.py

```diff
@@ -476,3 +486,67 @@ def delete_helper(self, team_name, user_id) -> ResponseTuple:
+        the teams on GitHub, this command can be called to fix things.
+        :return: return error message if user has insufficient permission level
+                 otherwise returns success messages with # of teams changed
+        """
+        num_changed = 0
+        num_added = 0
+        num_deleted = 0
+        modified = []
+        try:
+            command_user = self.facade.retrieve(User, user_id)
+            if not check_permissions(command_user, None):
+                return self.permission_error, 200
+            local_teams: List[Team] = self.facade.query(Team)
+            remote_teams: List[GithubTeam.Team] = self.gh.org_get_teams()
+            local_ids = dict((team.github_team_id, team)
+                             for team in local_teams)
+            remote_ids = dict((str(team.id),
+                               Team(str(team.id), team.name, ""))
+                              for team in remote_teams)
+
+            # remove teams not in github anymore
+            for local_id in local_ids:
+                if local_id not in remote_ids:
+                    self.gh.org_delete_team(local_id)
+                    num_deleted += 1
+                    modified.append(local_ids[local_id].get_attachment())
+
+            # add teams to db that are in github but not in local database
+            for remote_id in remote_ids:
+                if remote_id not in local_ids:
+                    self.facade.store(remote_ids[remote_id])
+                    num_added += 1
+                    modified.append(remote_ids[remote_id].get_attachment())
+                else:
+                    # and finally, if a local team differs, update it
+                    old_team = local_ids[remote_id]
+                    new_team = remote_ids[remote_id]
+                    if old_team.github_team_name != new_team.github_team_name\
+                            or old_team.members != new_team.members:
+
+                        # update the old team, to retain additional parameters
+                        old_team.github_team_name = new_team.github_team_name
+                        old_team.members = new_team.members
+                        self.facade.store(old_team)
+                        num_changed += 1
+                        modified.append(old_team.get_attachment())
+        except GithubAPIException as e:
+            logging.error("team refresh unsuccessful due to github error")
+            return "Refresh teams was unsuccessful with " \
+                   f"the following error: {e.data}", 200
+        except LookupError:
+            logging.error("team refresh unsuccessful due to lookup error")
+            return self.lookup_error, 200
+        status = f"{num_changed} teams changed, " \
+            f"{num_added} added, " \
+            f"{num_deleted} deleted. Wonderful."
+        ret = {'attachments': modified, 'text': status}
+        return jsonify(ret), 200
```

Comment:
@cheukyin699 so in what cases do we not return error code 200?

Reply:
We don't. We return nothing but 200s, according to [slack documentation][docs] on the slash command.

[docs]: https://api.slack.com/slash-commands#responding_with_errors

Reply:
hmm ok

Reply:
I think I understand. What should I call it instead?